src/comgr.cpp

```diff
@@ -94,6 +94,18 @@ namespace compiler {
 namespace lc {
 #define OCL_EARLY_INLINE 1
 
+namespace gcnasm {
+
+static void RemoveOptionsUnwanted(OptionList& list)
+{
+    list.erase(remove_if(list.begin(),
+                         list.end(),
+                         [&](const auto& option) { return StartsWith(option, "-mcpu="); }),
+               list.end());
```

Comment:
```suggestion
    auto is_mcpu = [&](const auto& option) { return StartsWith(option, "-mcpu=");
    auto remove_it = remove_if(list.begin(), list.end(), is_mcpu);
    list.erase(remove_it, list.end());
```

This stuff is usually easier to understand when split.